### DISCUSSION TITLE
Additional Console Commands, vim, nano, vi etc

### DISCUSSION THREAD

**Discussion by vwgtdmork** [louislam/dockge#482] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Would it be possible to run additional commands from within the console panel so that editing other files within the docker folder is possible.  It appears that the console is locked down to allow only a certain amount of commands.

### ✔️ Solution

I want to be able to be able to edit config files through the console section so that files mapped into the container volume can be edited.

for example.  

goto console section on page,
select an active docker compose and then perform a nano of a file that is being mapped into the container.

Currently if i want to edit a file that is being mapped into the container i have to log into the CLI.

**Comment by vishalkadam47** [comment]
add copy and paste option also

**Comment by CringleySDays** [comment]
Another thing I'd like to add is being able to use commands such as curl, mkdir too. It'd be very useful or allow any sorts of commands to be used and not to be limited at all.

**Reply by CringleySDays** [reply]
https://github.com/louislam/dockge/blob/d451e06e8428f1bf987595bf51d65f82241294cd/common/util-common.ts#L110

Seems like we'll need to change things from here?
